docker-compose.yml

```diff
@@ -1,5 +1,6 @@
 version: '2.3'
 
+# This value is used to name the `cli` image and refer to it, it can be most anything.
```

Comment:
this is not accurate statement. this value is used in multiple places, including Lagoon project name in open shift. Please either provide full explanation or remove this comment as it is confusing.

Reply:
Thanks. We are actually trying to work out what the constraints are on this name. So far the experience is "if i run multiple scaffolds locally i get a bunch of stuff with the same name". Do you know what the possible allowed names are?

Reply:
I guess it should say "this must match your projects as set by Finance, if you are not pushing this project to the Lagoon then it can be anything"